ohmac.sh

```diff
@@ -0,0 +1,853 @@
+			if [[ ${#COMMAND_LINE_ARGS} -eq 0 ]] && [[ $1 -eq 1 ]]; then
+				option="";
+				interactive_menu;
+			else
+				exit 1;
+			fi
+	exit 0;
+	esac
+}
+
+###################################################################
+function set_language {
+	# check for valid language selection
+	case "$OH_LANGUAGE" in 
+		en|fr|it|es|pt|ar) # TBD: language array direct check
+			# set localized database creation script
+			DB_CREATE_SQL="create_all_$OH_LANGUAGE.sql"
+			;;
+		*)
+			echo "Invalid language option: $OH_LANGUAGE. Exiting."
+			exit 1
+		;;
+	esac
+
+	# if $OH_SETTINGS is present set language
+	if [ -f ./$OH_DIR/rsc/$OH_SETTINGS ]; then
+		echo "Configuring OH language..."
+		######## $OH_SETTINGS language configuration
+		echo "Setting language to $OH_LANGUAGE in OH configuration file -> $OH_SETTINGS..."
+		sed -e "/^"LANGUAGE="/c"LANGUAGE=$OH_LANGUAGE"" -i ./$OH_DIR/rsc/$OH_SETTINGS
+		echo "Language set to $OH_LANGUAGE."
+	else 
+		echo ""
+		echo "Warning: $OH_SETTINGS file not found."
+	fi
+}
+
+###################################################################
+function install_brew {
+	if command -v brew &> /dev/null; then
+		echo "Homebrew is already installed."
+	else
+		# If brew is not installed, proceed with the installation
+		echo "Homebrew is not installed. Do you want to download it?"
+		get_confirmation;
+		/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+		
+		# Check if the installation was successful
+		if [ $? -eq 0 ]; then
+			echo "Homebrew has been installed successfully."
+		else
+			echo "An error occurred during Homebrew installation."
+		fi
+	fi
+}
+
+###################################################################
+function initialize_dir_structure {
+	# create directory structure
+	mkdir -p "./$TMP_DIR"
+	mkdir -p "./$LOG_DIR"
+	mkdir -p "./$DICOM_DIR"
+	mkdir -p "./$PHOTO_DIR"
+	mkdir -p "./$BACKUP_DIR"
+}
+
+###################################################################
+function install_db {
+    install_brew;
+    if brew list --formula | grep -q mariadb; then
+        echo "MariaDB is already installed."
+    else
+        echo "MariaDB is not installed. Do you want to download it?"
+        get_confirmation;
+        brew install mariadb;
+        brew services start mariadb >/dev/null;
+        create_db;
+    fi
+    return
+}
+
+###################################################################
+function test_db_connection {
+    # test if mysql client is available			
+    echo "Testing database connection..."	
+	
+
+	CMD="exit"		
+    if mysql -u $USER -h$DATABASE_SERVER -p$DATABASE_ROOT_PW -e "$CMD">/dev/null 2>&1; then
+        echo ">Database connection successfully established!"
+    else
+        echo "!Error: can't connect to database! Exiting."
+        exit 2
+    fi	 
+}
+
+###################################################################
+function import_db {
+    start_db;
+	
+    SCRIPTDIR="../../../openhospital-core/$SQL_DIR"
```

Comment:
Hello Max, I see, the syntax you've used is not suitable for the package we are distributing; you could try different approaches:

- cd to the right directory and then cd out (not nice, but it works :-)
- use ./$SQL_DIR/$DB_CREATE_SQL (as in the linux script)
- use /$OH_DIR/$SQL_DIR to use absolute path

You can try by "adding" ohmac.sh to an existing release package / directory structure and let me know !
Thank you !


./$SQL_DIR/$DB_CREATE_SQL